src/styles/blocks/cards/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;

```

Comment:
you don't have to hardcode these properties, they should be defined by content and paddings